### PR TITLE
Upgraded dependencies

### PR DESCRIPTION
**Pull request by davidrabinowitz**

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -2,6 +2,12 @@
 
 ## Next
 
+* BigQuery API has been upgraded to version 2.33.2
+* BigQuery Storage API has been upgraded to version 2.44.0
+* GAX has been upgraded to version 2.35.0
+* gRPC has been upgraded to version 1.58.0
+* Protocol Buffers has been upgraded to version 3.24.4
+
```
Comment (isha97) [inline CHANGES.md:10]: Do we need to document all version upgrades? I don;t see the dataproc, gcs version upgrades here

Comment (davidrabinowitz) [reply CHANGES.md:10]: We've been doing that for the past few years. It's also the common practice for our dependencies

**Review by isha97 — APPROVED** [review]

**Pull request merged**